.github/labeler.yml

```diff
@@ -1,6 +1,6 @@
 # https://github.com/marketplace/actions/labeler
 
-as-framework:
+kalix-proxy:
```

Comment:
Sounds strange to have a label `kalix-proxy` in the SDK, no?

Reply:
The purpose of this label is to be a "team" label that we use in the aggregated project board. I agree that kalix-proxy might not be the best, but I don't have a better suggestion.